acceptance/tests/fixtures/bases/job-client/privileged-scc-rolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Something went wrong with your rebase. You shouldn't have picked these up these openshift changes in your PR.